network/default.go

```diff
@@ -0,0 +1,336 @@
+package network
+
+import (
+	"sync"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/micro/go-micro/client"
+	"github.com/micro/go-micro/proxy"
+	"github.com/micro/go-micro/router"
+	pb "github.com/micro/go-micro/router/proto"
+	"github.com/micro/go-micro/server"
+	"github.com/micro/go-micro/transport"
+	"github.com/micro/go-micro/tunnel"
+	tr "github.com/micro/go-micro/tunnel/transport"
+	"github.com/micro/go-micro/util/log"
+)
+
+var (
+	// ControlChannel is the name of the tunnel channel for passing contron message
+	ControlChannel = "control-msg"
```

Comment:
Should we name this to `router` or something similar?

Reply:
That depends: are we going to be sending *ONLY router* adverts on this or are we going to re-use this particular `tunnel` channel for other "control" messages?

Reply:
Let's just call it control and reuse it for now. Probably keeps things simple.